target/linux/rtl838x/dts/rtl8380_tp-link_t2500g-10ts.dts

```diff
@@ -0,0 +1,302 @@
+/dts-v1/;
+
+#include "rtl838x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "tp-link,t2500g-10ts", "realtek,rtl838x-soc";
+	model = "TP-Link T2500-10TS Switch";
+
+	aliases {
+		led-boot = &led_sys;
+		led-failsafe = &led_sys;
+		led-running = &led_sys;
+		led-upgrade = &led_sys;
+	};	
+
+	chosen {
+		bootargs = "console=ttyS0,38400";
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+
+		led_sys: sys {
+			label = "t2500g-10ts:green:sys";
+			gpios = <&gpio0 148 GPIO_ACTIVE_HIGH>;
+		};
+		led_lan1ge: lan1ge {
```

Comment:
These DT labels don't seem to be required? Just lan1g should be enough?

Reply:
Indeed. Removed.